client-runtime/src/main/java/com/microsoft/rest/v2/RestResponse.java

```diff
@@ -39,63 +34,30 @@ public RestResponse(int statusCode, THeaders headers, Map<String, String> rawHea
     }
 
     /**
-     * The status code of the HTTP response.
      * @return The status code of the HTTP response.
      */
     public int statusCode() {
         return statusCode;
     }
 
     /**
-     * The deserialized headers of the HTTP response.
      * @return The deserialized headers of the HTTP response.
      */
     public THeaders headers() {
         return headers;
     }
 
     /**
-     * The raw HTTP response headers.
      * @return A Map containing the raw HTTP response headers.
      */
     public Map<String, String> rawHeaders() {
         return rawHeaders;
     }
 
     /**
-     * The deserialized body of the HTTP response.
      * @return The deserialized body of the HTTP response.
      */
     public TBody body() {
         return body;
     }
-
-    /**
-     * Closes the content stream associated with this RestResponse, if any.
-     */
-    public void close() {
```

Comment:
This is now handled only in concrete responses where we know the body is a `Flowable<ByteBuffer>`.